lib/tests/formulae.rb

```diff
@@ -38,11 +38,16 @@ def run!(args:)
         # #run! modifies `@testing_formulae`, so we need to track this separately.
         @testing_formulae_count = @testing_formulae.count
         perform_bash_cleanup = @testing_formulae.include?("bash")
+        @tested_formulae_count = 0
 
         sorted_formulae.each do |f|
           formula!(f, args:)
           verify_local_bottles
           puts
+          info_header "Tests progress"
+          puts "Testing formulae count:   #{@testing_formulae_count}"
+          puts "Tested formulae count:    #{@tested_formulae_count}"
+          puts "Remaining formulae count: #{@testing_formulae_count - @tested_formulae_count}"
```

Comment:
Could this get slimmed into:
- a single line
- avoid printing it if `@testing_formulae_count` == 1 (or maybe e.g. <3 or similar?)